src/core_seaice/shared/mpas_seaice_forcing.F

```diff
@@ -1685,21 +1698,35 @@ end subroutine init_data_iceberg_forcing
 
   subroutine data_iceberg_forcing(&
        streamManager, &
-       domain, &
-       simulationClock)
+       domain)
 
     type (MPAS_streamManager_type), intent(inout) :: streamManager
 
     type (domain_type) :: domain
 
-    type (MPAS_clock_type) :: simulationClock ! dc is this being used?
+    ! for debugging
+    ! type (MPAS_time_type) :: currentForcingTime
+    ! character(len=strKIND) :: currentForcingTimeStr
+
+    logical, pointer :: &
+         config_do_restart
```

Comment:
@akturner This argument is unused in this subroutine - I'll remove it with the next commit with better comments on the debugging lines.